src/runtime-rs/crates/agent/src/log_forwarder.rs

```diff
@@ -76,7 +77,11 @@ impl LogForwarder {
                         }
                     }
                     Err(err) => {
-                        warn!(logger, "connect agent vsock failed: {:?}", err);
+                        trace!(
```

Comment:
Why here downgrade the log level to trace ?  if we keep it as warn, can we get the message `"connect agent vsock failed..."` ?

Reply:
This log is in the loop, and the level is too high. As a result, you still see lots of `connect agent vsock...`. I think it's good to downgrade to trace level.

Reply:
I print two logs, one before the loop, one after a successful connection. If people are aware of failure to connect, they could enable trace level logging to investigate the root cause.